### ISSUE TITLE
RN 0.40.0 iOS Won't Build: ld: 102 duplicate symbols for architecture arm64

### ISSUE THREAD

**Issue by dwilt** [OneSignal/react-native-onesignal#199]
Things were working perfectly before trying to upgrade to React Native 0.40.0. I was able to get all of our other dependencies working but this one and I'm not sure what's wrong.

I tried uninstalling the plugin completely, then cleaning the project, and the reinstalling and I keep ending up with this error while trying to build:

![image](https://cloud.githubusercontent.com/assets/1065539/23096683/5ec23a04-f5d6-11e6-9c8a-376e5a95ae66.png)

package.json
```
"react-native": "0.40.0",
"react-native-onesignal": "^3.0.3",
```

In my `Podfile`, I'm using version `2.3.5` of the OneSignal SDK

**Comment by Ouadie** [comment]
Same issue for me

**Comment by jkasten2** [comment]
@dwilt @Ouadie As of version 3.0.3 the OneSignal iOS native SDK is now part of it. You will need to remove  the Cocoapod, Carthage, manually added copy after updating.

To remove it from cocoapods remove OneSignal from your `Podfile` and run `pod install`.

**Comment by dwilt** [comment]
@jkasten2 I got this working now but I'm not sure what I changed. I completely uninstalled `react-native-onesignal` via `npm`. I removed any reference of it from my Android and iOS files. I unlinked it with the library and also removed it from the `Podfile` as @jkasten2 suggested.

I reinstalled everything, cleaned my project (cmd-shift-k), rebuilt (cmd-r) and it worked. Not sure what else to say. These iOS projects can be so annoying.
